demo/scripts/startFPCAuctionNetwork.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+#
+# Copyright IBM Corp All Rights Reserved
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+export DEMO_SCRIPTS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+export FPC_ROOT=${DEMO_SCRIPTS_DIR}/../..
+export DEMO_ROOT=${DEMO_SCRIPTS_DIR}/..
+
+# SCRIPT_DIR is the docker compose script dir that needs to be defined to source environment variables from the FPC Network
+export SCRIPT_DIR=${FPC_ROOT}/utils/docker-compose/scripts
+. ${SCRIPT_DIR}/lib/common.sh
+
+# Cleanup any previous iterations of the demo
+"${SCRIPT_DIR}/teardown.sh" --clean-slate
+
+# Generate the necessary credentials and start the FPC network
+"${SCRIPT_DIR}/generate.sh"
+"${SCRIPT_DIR}/start.sh"
+
+# Install and Instantiate Auction Chaincode
+"${DEMO_SCRIPTS_DIR}/installCC.sh"
+
+# Register Users
+"${DEMO_ROOT}/client/backend/fabric-gateway/registerUsers.sh"
+
+# Build Auction Client Backend
+pushd  "${DEMO_ROOT}/client/backend/fabric-gateway"
```

Comment:
Couldn't we capture that inside docker-compose file? If so, would be nicer/easier?

Reply:
I have to look into that. I have never built images using docker-compose

Reply:
it's pretty straight forward, .e.,g below an example from PDO:
```
..... 
 pdo-build:
    image: pdo-sim-build
    build:
      context: .
      dockerfile: Dockerfile.pdo-build
      args:
        PDO_REPO_URL: ${PDO_REPO_URL:-https://github.com/hyperledger-labs/private-data-objects.git}
        PDO_REPO_BRANCH: ${PDO_REPO_BRANCH:-master}
        SGX_MODE: SIM
        PDO_INTERPRETER: ${PDO_INTERPRETER:-gipsy}
....
```

Reply:
perfect! i was struggling finding non trivial build docker-compose files in the docker docs.

Reply:
docker images have been updated so that the auction client and auction frontend are built if not already there when using docker-compose to start the services.